src/pudl/transform/classes.py

```diff
@@ -817,7 +817,12 @@ def drop_invalid_rows(df: pd.DataFrame, params: InvalidRows) -> pd.DataFrame:
     # Create a boolean mask selecting the ROWS where NOT ALL of the columns we
     # care about are invalid (i.e. where ANY of the columns we care about contain a
     # valid value):
-    mask = ~cols_to_check.isin(params.invalid_values).all(axis="columns")
+
+    # for some reason, a simple cols_to_check.isin(params.invalid_values) fails
```

Comment:
TODO: add specific error + more context from [Slack](https://catalystcooperative.slack.com/archives/C01UQPVETGD/p1694202142945359)

```
TypeError: boolean value of NA is ambiguous
```

Basically this is a fake .isin() because that was running into issues with NA values that we couldn't drop ahead of time.